### DISCUSSION TITLE
2021/building-and-publishing-debrpm-packages-with-goreleaser-and-gemfury/

### DISCUSSION THREAD

**Discussion by giscus** [hellt/netdevops.me#16] [Show and tell]
# 2021/building-and-publishing-debrpm-packages-with-goreleaser-and-gemfury/

Applying DevOps to networks.

https://netdevops.me/2021/building-and-publishing-debrpm-packages-with-goreleaser-and-gemfury/

**Comment by maidul98** [comment]
Do you not forget to change the username of 
```
https://{{ .Env.FURY_TOKEN }}@push.fury.io/netdevops/
```

Change `netdevops` with your username. The article forgot to mention this. https://gemfury.com/help/upload-packages#cURL&_ga=2.56888234.1885891812.1668374610-1995395227.1668374610

**Reply by hellt** [reply]
thanks @maidul98, I have updated this
